data/datafiles.json

```diff
@@ -619,8 +619,14 @@
 	    "filename": "ramses_extra_fields",
 	    "size": "220 MB",
 	    "url": "http://yt-project.org/data/ramses_extra_fields.tar.gz"
+	},
+	{
+	    "code": "RAMSES",
+	    "description": "Hydro run, unzip to 26MB with extra particle fields (temperature + family)",
+	    "filename": "ramses_extra_fields_small",
+	    "size": "15 MB",
+	    "url": "http://yt-project.org/data/ramses_custom_fields.tar.gz"
```

Comment:
It would be less confusing if the name were `ramses_extra_fields_small` so I'd go for that.